tests/webtests/ServiceOverviewTest.php

```diff
@@ -111,4 +172,23 @@ public function test_service_overview_shows_message_when_no_service_selected()
         $this->assertContains('No service selected', $response->getContent());
         $this->assertContains('Please select a service', $response->getContent());
     }
+
+
+    private function rowsToArray(Crawler $crawler)
```

Comment:
Nice touch, maybe add this to the abstract WebTest class? might be usefull for other similar tests!

Reply:
I don't like one god webtestcase, but feel free to reuse